lms/static/sass/discussion/_discussion.scss

```diff
@@ -4,17 +4,13 @@
 
 body.discussion {
 
-  .course-tabs .right {
-    @include float(right);
-  }
-
   .edit-post-form {
     @include clearfix();
     box-sizing: border-box;
     width: 100%;
 
     h1 {
-      font-size: 20px;
+      font-size: $forum-x-large-font-size;
```

Comment:
Are these styles meant to be different from the rest of the platform (UXPL) styles? I kind of think we should use the same sizings where possible to prevent getting to the same state things are in now, in the future.